frontend/src/lib/components/AccessControlAction.tsx

```diff
@@ -0,0 +1,33 @@
+import { useValues } from 'kea'
+
+import { accessControlLogic } from '~/layout/navigation-3000/sidepanel/panels/access_control/accessControlLogic'
+
+import { WithAccessControl } from '../../types'
+
+interface AccessControlActionProps {
+    children: (props: { disabled: boolean; disabledReason: string | null }) => React.ReactElement
+    userAccessLevel?: WithAccessControl['user_access_level']
+    requiredLevels: WithAccessControl['user_access_level'][]
```

Comment:
Feels weird that you've opted for listing the levels instead of having a "minRequiredLevel" like we do in the backend. Is there a reason for this that I've missed?

Reply:
You're right, it was an oversight - will update.

Reply:
Roles are seperate from levels. 

Roles are groups of users.

Levels are like view, editor, admin, etc. which are the related to the permissions that user or role has on the resource. 
	- organizations have members, admins, owners
	- projects have members, admins
	- resources has viewer, editor